spectator/meter/max_gauge.go

```diff
@@ -0,0 +1,36 @@
+package meter
+
+import (
+	"fmt"
+	"github.com/Netflix/spectator-go/spectator/writer"
+)
+
+// MaxGauge represents a value that is sampled at a specific point in time. One
```

Comment:
At some point, we are going to consolidate all of the Spectatord and thin client documentation at the main Atlas Docs site:

https://netflix.github.io/atlas-docs/spectator/lang/go/usage/

It will take some time though. Main advice here is not to over-index on documenting the basic meter types and usage here - the main spectatord site and the spectator-py site have lots of that already that we can use as a reference. I would keep documentation updates focused on the uniqueness that is the Go implementation.

Reply:
Sounds good 👍
This is basically the same doc that exists for Gauge, just slightly adapted.